### PR TITLE
Add MainButton

### PR DESCRIPTION
**Pull request by sepehrheravi**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+import LandingScreen from "./screens/LoginScreens/LandingScreen";
 
 export default function App() {
   return (
-    <LandingScreen/>
+    // <LandingScreen/>
     // <LoginScreen/>
     // <LoginVerificationScreen/>
+
+    //Testing Button
```
Comment (gsarthakdev) [inline App.js:12]: Remove line of code.

File: App.js [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+    <View style={styles.container}>
```
Comment (gsarthakdev) [inline App.js:13]: Remove line of code.

File: App.js [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
+      <MainButton>Sign In</MainButton>
```
Comment (gsarthakdev) [inline App.js:14]: Remove line of code.

File: App.js [hunk 4]
```diff
@@ -1,20 +1,26 @@
...
+    </View>
```
Comment (gsarthakdev) [inline App.js:15]: Remove line of code.

File: App.js (hunk 1 above)
Comment (gsarthakdev) [inline App.js:12]: Please remove the following code: 
```   
    //Testing Button
    <View style={styles.container}>
      <MainButton>Sign In</MainButton>
    </View>

File: components/MainButton.js [hunk 5]
```diff
@@ -1 +1,37 @@
...
\ No newline at end of file
+import React from "react";
+import { View, StyleSheet, Pressable, Text, PixelRatio } from "react-native";
+import { GlobalStyles } from "../constants/styles";
+
+function MainButton({ children }) {
+  return (
+    <View style={styles.container}>
+      <Pressable
+        style={({ pressed }) => [{ opacity: pressed ? 0.75 : 1 }]}
```
Comment (gsarthakdev) [inline components/MainButton.js:9]: Change to: 

`style={({pressed}) => pressed && {opacity: 0.75}}`

File: components/MainButton.js [hunk 6]
```diff
@@ -1 +1,37 @@
...
+        <View style={styles.button}>
+          <Text style={styles.text}>{children}</Text>
+        </View>
+      </Pressable>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: { flex: 1, justifyContent: "center", alignItems: "center" },
```
Comment (gsarthakdev) [inline components/MainButton.js:21]: `container` style is not needed for components, such as buttons. 

Please remove `container` style from the StyleSheet.

File: components/MainButton.js [hunk 7]
```diff
@@ -1 +1,37 @@
-//opacity: 0.75
\ No newline at end of file
+import React from "react";
+import { View, StyleSheet, Pressable, Text, PixelRatio } from "react-native";
+import { GlobalStyles } from "../constants/styles";
+
+function MainButton({ children }) {
+  return (
+    <View style={styles.container}>
```
Comment (gsarthakdev) [inline components/MainButton.js:7]: When creating custom buttons with Pressable, a wrapper `<View> </View>` is not needed, please remove it.

File: components/MainButton.js [hunk 8]
```diff
@@ -1 +1,37 @@
...
+    height: 50,
+    backgroundColor: GlobalStyles.colors.purpleBackground,
+    borderRadius: 30,
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  text: {
+    fontWeight: "700",
+    fontSize: 18,
+    color: "#FFFFFF",
```
Comment (gsarthakdev) [inline components/MainButton.js:33]: Please change to `color: "white"`

File: components/MainButton.js [hunk 9]
```diff
@@ -1 +1,37 @@
...
+  container: { flex: 1, justifyContent: "center", alignItems: "center" },
+  button: {
+    width: 315,
+    height: 50,
+    backgroundColor: GlobalStyles.colors.purpleBackground,
+    borderRadius: 30,
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  text: {
```
Comment (gsarthakdev) [inline components/MainButton.js:30]: Change style name to `label`.

File: components/MainButton.js [hunk 10]
```diff
@@ -1 +1,37 @@
...
+    fontWeight: "700",
+    fontSize: 18,
+    color: "#FFFFFF",
+  },
+});
+
+export default MainButton;
```
Comment (gsarthakdev) [inline components/MainButton.js:37]: Move this `export default MainButton;` statement to Line 19, right before you declare `const styles =`

File: constants/styles.js [hunk 11]
```diff
@@ -1,5 +1,6 @@
 export const GlobalStyles = {
   colors: {
-    lightModeBackground: "#F5F5F5"
-  }
-}
\ No newline at end of file
+    lightModeBackground: "#F5F5F5",
+    purpleBackground: "#7141FA",
```
Comment (gsarthakdev) [inline constants/styles.js:4]: Change `purpleBackground` to `primaryPurple`

**Review by gsarthakdev — CHANGES_REQUESTED** [review]
Please review the comments on all three files and make the changes requested.

**Review by sepehrheravi — COMMENTED** [review]
The changes were made per your request.

**Review by gsarthakdev — APPROVED** [review]
Approved `Add MainButton`

**Pull request merged**